channels/api.py

```diff
@@ -978,6 +1043,8 @@ def add_moderator(self, moderator_name, channel_name):
         except APIException as ex:
             if ex.error_type != "ALREADY_MODERATOR":
                 raise
+        update_user_role(channel_name, ROLE_MODERATORS, user)
```

Comment:
I feel like it's not clear on reading this that it is adding the user, I think it might be better to be more explicit by having a method for to add and a method to remove